public/script.js

```diff
@@ -69,6 +69,8 @@ class Users {
             resolve();
         });
     }
+
+    //refreshAccessToken() {}
```

Comment:
Should this be included if it's commentted out?

Reply:
I left it there as a reminder to implement it later potentially. Right now the way participants are loaded in from local storage they are no longer User Objects, but just Objects with user data not methods, so if I find a way to rework that I would like to just be able to call refresh token on a user.